scripts/default/integtest.sh

```diff
@@ -87,9 +87,17 @@ then
   SNAPSHOT="false"
 fi
 
+ IFS='.' read -ra version_array <<< "$OPENSEARCH_VERSION"
+
+
 if [ -z "$CREDENTIAL" ]
 then
-  CREDENTIAL="admin:admin"
+  # Starting in 2.12.0, security demo configuration script requires an initial admin password
+  if (( ${version_array[0]} > 2 || (${version_array[0]} == 2 && ${version_array[1]} >= 12) )); then
+    CREDENTIAL="admin:myStrongPassword123!"
+  else
+    CREDENTIAL="admin:admin"
+  fi
```

Comment:
1. You are changing IFS without resetting it back to original
2. Why not use awk or cut with separator like on line 95/103?

Reply:
Fixed as discussed with this commit: [7480dde](https://github.com/opensearch-project/opensearch-build/pull/4302/commits/7480dde2a05f95ac25de7c9644b7197239e1875f)